booth.spec

```diff
@@ -1,5 +1,11 @@
 %bcond_with html_man
 
+%if 0%{?fedora} > 18 || 0%{?centos} > 6 || 0%{?rhel} > 6
```

Comment:
Nope, that should be correct:
http://rpm.org/wiki/PackagerDocs/ConditionalBuilds

We want "without glue" by default for those Fedora, etc. distros, and "with glue" otherwise.
I think I know where's the issue, we need to pass `[other-libraries]` to `AC_CHECK_LIB` for `plumb` library. Will try to figure it out.